quacc/custodian/qchem.py

```diff
@@ -0,0 +1,84 @@
+"""
+Custodian handlers for QChem
+"""
+from __future__ import annotations
+
+from custodian import Custodian
+from monty.dev import requires
+
+from quacc import SETTINGS
+
+try:
+    import openbabel as ob
+except ImportError:
+    ob = None
+
+
+@requires(
+    ob,
+    "Openbabel must be installed. Try conda install -c conda-forge openbabel",
+)
+def run_custodian(
+    qchem_cmd: str = SETTINGS.QCHEM_CMD,
+    qchem_max_cores: int = SETTINGS.QCHEM_MAX_CORES,
+    qchem_calc_loc: str = SETTINGS.QCHEM_CALC_LOC,
+    qchem_custodian_max_errors: int = SETTINGS.QCHEM_CUSTODIAN_MAX_ERRORS,
+    qchem_custodian_handlers: list[str] = SETTINGS.QCHEM_CUSTODIAN_HANDLERS,
+) -> None:
+    """
+    Function to run QChem Custodian
+
+    Parameters
+    ----------
+    qchem_cmd
+        Q-Chem command. Defaults to "qchem" in settings.
+    qchem_max_cores
+        Maximum number of cores to use for the Q-Chem calculation. Defaults to 32 in settings.
+    qchem_calc_loc
+        Compute-node local scratch directory in which Q-Chem should perform IO. Defaults to /tmp in settings.
+    qchem_custodian_max_errors
+        Maximum number of errors to allow before stopping the run. Defaults to 5 in settings.
+    qchem_custodian_handlers
+        List of handlers to use in Custodian. See settings for list.
+
+    Returns
+    -------
+    None
+    """
+    # Adapted from atomate.qchem.firetasks.run_calc
+
+    from custodian.qchem.handlers import QChemErrorHandler
+    from custodian.qchem.jobs import QCJob
+
+    # Handlers for Q-Chem
+    handlers_dict = {
+        "QChemErrorHandler": QChemErrorHandler(),
+    }
+
+    handlers = []
+    for handler_flag in qchem_custodian_handlers:
+        if handler_flag not in handlers_dict:
+            raise ValueError(f"Unknown Q-Chem error handler: {handler_flag}")
```

Comment:
@arosen93 Since there's only one Q-Chem error handler (and I can't really see that ever changing), do we think that this approach is really necessary here?

Reply:
If `qchem_custodian_handlers` is an input kwarg, then yeah we should check to make sure the user provided a valid one. However, if that's the only error handler, we can just drop the kwarg entirely and force that as the only handler in the function itself.

Reply:
Dropped the kwarg and global setting entirely